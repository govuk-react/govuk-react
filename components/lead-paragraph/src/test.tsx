import React from 'react';
import ReactDOM from 'react-dom';
import { mount } from 'enzyme';

import LeadParagraph from '.';

describe('LeadParagraph', () => {
  const example = 'example';
  const wrapper = <LeadParagraph>{example}</LeadParagraph>;

  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<LeadParagraph>Example</LeadParagraph>, div);
  });

  it('matches wrapper snapshot', () => {
    expect(mount(wrapper)).toMatchSnapshot('wrapper mount');
  });
});
