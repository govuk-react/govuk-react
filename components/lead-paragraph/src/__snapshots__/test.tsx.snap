// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LeadParagraph matches wrapper snapshot: wrapper mount 1`] = `
.c0 {
  color: #0b0c0c;
  font-family: "nta",Arial,sans-serif;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  font-weight: 400;
  font-size: 18px;
  line-height: 1.1111111111111112;
  margin-top: 0;
  margin-bottom: 20px;
}

@media print {
  .c0 {
    color: #000;
  }
}

@media print {
  .c0 {
    font-size: 18px;
    line-height: 1.15;
  }
}

@media only screen and (min-width:641px) {
  .c0 {
    font-size: 24px;
    line-height: 1.25;
  }
}

@media only screen and (min-width:641px) {
  .c0 {
    margin-bottom: 30px;
  }
}

<LeadParagraph>
  <p
    className="c0"
  >
    example
  </p>
</LeadParagraph>
`;
